tests/pebble.rs

```diff
@@ -335,6 +335,19 @@ impl Environment {
         Ok(roots)
     }
 
+    async fn request(&self, path: &str, body: &impl Serialize) -> Result<(), Box<dyn StdError>> {
```

Comment:
I think `request` as a name might be too short to convey that it's always using the challtestsrv as a server (vs, say, a request to Pebble, or Pebble's own management interface).

WDYT about `challtestsrv_request`? or `challenge_management_request`? A bit wordy, but at least clear.